### ISSUE TITLE
C to F

### ISSUE THREAD

**Issue by ejmorgan55** [txoof/epd_display#55]
Looking in the config files; cannot find where to convert the Met No from Celsius to Fahrenheit? Can you point me in the right direction? Thanks.

**Comment by ejmorgan55** [comment]
Sorry; I found it; right in the readme file. Added the line and I now have F rather than C. Thanks.

**Comment by txoof** [comment]
The following lines in the `paperpi.ini` file can be added to control units:
```
'temp_units': 'celsius' or 'fahrenheit' (`str`), #optional
'rain_units': 'mm' or 'inch' (`str`), #optional
```

**Comment by txoof** [comment]
resolved with commit 59937918563fc88d130363c2f46a3c801edfd8b5
